tests/cases/tm2tm/scripts/test-service

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+: <<'END_COMMENT'
+* relay-interval = 20s
+* src-relay-optimize-interval = 25s
+* src-relay-optimize-count = 3
+* dst-relay-optimize-interval = 25s
+* dst-relay-optimize-count = 3
+
+- relay service [packets = 0, time = 0] -> skip
+    - sleep 4
+    - 
```

Comment:
I like this `sleep N` style. Empty lines are meaningless and should be removed now.